### PR TITLE
Updates for JWebbinar 8

### PR DESCRIPTION
**Pull request by cslocum**
- Eliminate jdaviz env/kernel, move those packages into jwebbinar env
- Freeze docker-stacks revision for use if USE_FROZEN=1
- Misc. minor cleanup

TODO:
- [x] Obtain notebooks
- [x] Add notebook CI tests
- [x] Set default notebook kernel to jwebbinar
- [ ] Manually test notebooks, with special attention to jdaviz functionality

### CODE REVIEW

File: tools/image-base [hunk 1]
```diff
@@ -19,14 +19,25 @@ docker build --progress plain --tag ${OWNER}/ubuntu-certs --file Dockerfile.cert
...
-   cd docker-stacks
+    git clone https://github.com/jupyter/docker-stacks.git
+    cd docker-stacks
 else
     cd docker-stacks
     git fetch origin
     git rebase origin/master
 fi
 
+# checkout the appropriate docker-stacks revision if using frozen specs;
```
Comment (jaytmiller) [inline tools/image-base:30]: If desired,  you can streamline this some using ${IMAGE_DIR} to replace: ${JUPYTERHUB_DIR}/deployments/$DEPLOYMENT_NAME/image

**Review by jaytmiller — APPROVED** [review]
Nice!

File: tools/image-base (hunk 1 above)
Comment (cslocum) [reply tools/image-base:30]: Done.

File: tools/image-base [hunk 2]
```diff
@@ -19,14 +19,25 @@ docker build --progress plain --tag ${OWNER}/ubuntu-certs --file Dockerfile.cert
...
+# otherwise, collect and stash revision
+env_frozen_dir="${IMAGE_DIR}/env-frozen"
+mkdir -p $env_frozen_dir
+frozen_hash_spec="$env_frozen_dir/docker-stacks.hash"
+if [[ USE_FROZEN -eq 1 ]]; then
```
Comment (jaytmiller) [inline tools/image-base:35]: Should this be $USE_FROZEN?

**Review by jaytmiller — CHANGES_REQUESTED** [review]

**Review by jaytmiller — APPROVED** [review]
Nice!

**Pull request merged**